README.md

```diff
@@ -289,7 +289,7 @@ yo chisel:page "Home" "About Us" "Contact Us" "News"
 When you have the basic setup done, you can start development. To re-compile Twig, SCSS and JavaScript files in real time you can use default task. Type
 
 ```bash
-gulp
+npm run dev
```

Comment:
Btw. I suggest doing README updates in a separate PRs after the new functionality is released. Now README says you should use `npm run dev`, etc. but I guess it won't work with the current version. Anyway, going to make a new release now so it does.